MVC/view/CLIrefactor.py

```diff
@@ -23,6 +25,9 @@ def __init__(self):
         #variable to store user letters into a list for displaying a honeycomb.
         self.displayLetters = []
         self.check = False
+        self.b4commands = ["!newpuzzle","!loadpuzzle","!help","!exit"]
+        self.commands = ["!newpuzzle","!loadpuzzle","!help","!exit","!showpuzzle","!showfoundwords","!shuffle","!savepuzzle","!showstatus"]
```

Comment:
Do these show with tab in the order you have them listed here? If so, should make it consistent with the order in the !help section.

Reply:
one sec, changing it.